test/test_core.py

```diff
@@ -6921,7 +6919,7 @@ def test_EXPORTED_RUNTIME_METHODS(self):
   def test_dyncall_specific(self, *args):
     if self.get_setting('MEMORY64'):
       self.skipTest('not compatible with MEMORY64')
-    if self.get_setting('WASM_BIGINT'):
+    if self.get_setting('WASM_BIGINT', 1) != 0:
```

Comment:
This broke because `get_setting` only returns settings modified by `set_settings`, not default ones. This logic checks whether WASM_BIGINT hasn't been set to 0, but I'm not sure if there's a better way.

Reply:
Can you not just do `self.get_setting('WASM_BIGINT') != 0`?.   I think it it has not been set at all this will return `None`.

Reply:
Hm yes but actually neither one of those works in the wasm2js test suite, where WASM_BIGINT is set to 0 by the code in link.py rather than test code.

Reply:
I guess you need to test for wasm2js here then since that is a way if implicitly disabling WASM_BIGINT?